packages/components/src/__future__/Tag/Tag.tsx

```diff
@@ -1,60 +1,30 @@
 import React, { ReactNode, HTMLAttributes } from "react"
 import classNames from "classnames"
 import { OverrideClassName } from "~types/OverrideClassName"
-
-export type TagColors =
-  | "gray"
-  | "blue"
-  | "green"
-  | "yellow"
-  | "orange"
-  | "red"
-  | "purple"
+import { TagColors } from "./types"
+import styles from "./Tag.module.scss"
 
 export interface TagProps
   extends OverrideClassName<HTMLAttributes<HTMLSpanElement>> {
   children: ReactNode
   color?: TagColors
-  Icon?: React.ReactElement
+  icon?: React.ReactElement
 }
 
 export const Tag = ({
   children,
   classNameOverride,
-  Icon,
+  icon,
   color = "gray",
 }: TagProps): JSX.Element => (
   <span
-    className={classNames(
-      "inline-flex items-center px-8 py-[5px] gap-4 rounded-[28px] font-family-paragraph text-paragraph-sm font-weight-paragraph leading-paragraph-sm whitespace-nowrap",
-      {
-        ["bg-gray-200 text-purple-800"]: color === "gray",
-        ["bg-blue-100 text-blue-700"]: color === "blue",
-        ["bg-green-100 text-green-700"]: color === "green",
-        ["bg-yellow-100 text-yellow-700"]: color === "yellow",
-        ["bg-orange-100 text-orange-700"]: color === "orange",
-        ["bg-red-100 text-red-700"]: color === "red",
-        ["bg-purple-100 text-purple-700"]: color === "purple",
-      },
-      classNameOverride
-    )}
+    className={classNames(styles.container, styles[color], classNameOverride)}
```

Comment:
Usually we'd use the component name for the main classname :) It's part of the template, but I suppose it was removed in the contribution 😅 

```suggestion
    className={classNames(styles.tag, styles[color], classNameOverride)}
```